cmd/generate/config/rules/nuget.go

```diff
@@ -0,0 +1,29 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+	"regexp"
+
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func NugetConfigPassword() *config.Rule {
+	r := config.Rule{
+		Description: "Identified a password within a Nuget config file, potentially compromising package management access.",
+		RuleID:      "nuget-config-password",
+		Regex:       regexp.MustCompile(`(?i)(<add key=\"(ClearTextPassword)|(Password)\"\s*)(value=\"([A-Za-z0-9]+)\"\s*)(/>)`),
+		Path:        regexp.MustCompile(`(?i)nuget\.config$`),
+		SecretGroup: 5,
```

Comment:
The regex can be simplified to have only one capture group, then `SecretGroup` isn't necessary:
```suggestion
		Regex:       regexp.MustCompile(`<add key=\"(?:(?:ClearText)?Password)\"\s*value=\"([A-Za-z0-9]+)\"\s*/>`),
		Path:        regexp.MustCompile(`(?i)nuget\.config$`),
```

It may also help to add a minimum length (e.g., `[A-Za-z0-9]{8,}`) and entropy check.

Reply:
1. cool, thanks
2. added length and entropy
3. I prefer having case-insensitive, you never know :smile: 
4. I changed to cover all kind of characters